400-claims.md

```diff
@@ -96,10 +96,11 @@ The fields above are defined as follows:
   - `action` (REQUIRED): The name of the action that last caused this claim to be updated. This may be any of the built-in actions (`install`, `upgrade`, `uninstall`) as well as any custom actions as defined in the bundle descriptor. The special name `unknown` MAY be used in the case where the CNAB Runtime cannot determine the action name of a claim.
   - `message` (OPTIONAL): A human-readable string that communicates the outcome. Error messages MAY be included in `failure` conditions.
   - `status` (REQUIRED): Indicates the status of the last phase transition. Valid statuses are:
-    - `failure`: failed before completion
-    - `pending`: in progress. This should only be used if the invocation container MUST exit before it can determine whether all operations are complete. Note that `pending` is a _long term status_ that indicates that the installation's final state cannot be determined by the system. For this reason, it should be avoided. When used, `pending` should be considered a temporary status, and the runtime SHOULD work to resolve this to either `failure` or `success`.
-    - `unknown`: the state is unknown. This is an error condition.
-    - `success`: completed successfully
+    - `failure`: Failed before completion.
+    - `pending`: Execution has been requested and has not begun.
```

Comment:
>  When used, `pending` should be considered a temporary status, and the runtime SHOULD work to resolve this to either `failure` or `success`.

I like this new `running` state, but I think we should keep the part enforcing that this status (and also `pending`) is only temporary and that the tools should resolve it .